todoist/scenes.py

```diff
@@ -28,6 +31,60 @@ def from_request(cls, request: Request, intent_name: str):
             return cls.TOMORROW
 
 
+class Time:
+    @classmethod
+    def from_request_slot(cls, request: Request, intent_name: str, time_key: str) -> Optional[date]:
+        time_from_intent = request.intents[intent_name].get('slots', {}).get(time_key, {})
+
+        if time_from_intent['type'] != 'YANDEX.DATETIME':
```

Comment:
If the user doesn't mention a date, there will be no 'type' key in the intent.

Reply:
maybe consider today as a default?